terraso_backend/apps/soil_id/graphql/soil_data.py

```diff
@@ -235,11 +235,20 @@ class Input:
         sodium_adsorption_ratio_enabled = graphene.Boolean()
         soil_structure_enabled = graphene.Boolean()
 
+        new_depth_interval = graphene.Field(DepthIntervalInput)
```

Comment:
question: what is the intention/use of the `new_depth_interval` arg? it seems to be for mutating an existing depth interval's start and end values, but i don't understand why we would want to do that